src/lib.js

```diff
@@ -253,11 +257,10 @@ export default class Migrator {
           migrationName = migrationToImport.slice(timestampSeparatorIndex + 1, migrationToImport.lastIndexOf('.'));
 
         this.log(`Adding migration ${filePath} into database from file system. State is ` + `DOWN`.red);
-        const createdMigration = MigrationModel.create({
+        return MigrationModel.create({
```

Comment:
Oh, and BTW, all the other instances async work - so it' something about the context of this location, perhaps b/c it's in a an anonymous function, but not 100% sure.